### PR TITLE
Fix/update stories examples

### PR DESCRIPTION
**Pull request by zcernigoj**
- cleaned up storybook stories
  - correct `BBox` and `...Layer` initializations 
    - also moved bbox vars out of separate stories so there are only 2 vars with bbox (one in EPSG:4326 and one in EPSG:3857) defined at the top of each storybook file 
  - added `findFlyovers()` for some layers

- added `stories` folder to prettier, so the code in stories is also prettified

### CODE REVIEW

File: stories/index.stories.js [hunk 1]
```diff
@@ -323,7 +318,7 @@ function renderTilesList(containerEl, list) {
   });
 }
 
-async function setAuthTokenWithOAuthCredentials () {
+async function setAuthTokenWithOAuthCredentials() {
```
Comment (dgostencnik) [inline stories/index.stories.js]: `setAuthTokenWithOAuthCredentials` and `renderTilesList` should probably be extracted as they are being copied into each story. feel free to ignore.

File: stories/landsat8.aws.stories.js [hunk 2]
```diff
@@ -7,23 +7,23 @@ import {
...
-};
+  throw new Error('LANDSAT8_LAYER_ID environment variable is not defined!');
+}
 
 const instanceId = process.env.INSTANCE_ID;
 const layerId = process.env.LANDSAT8_LAYER_ID;
 const bbox = new BBox(CRS_EPSG3857, 1487158.82, 5322463.15, 1565430.34, 5400734.67);
-const bbox4326 = new BBox(CRS_EPSG4326, 11.9, 42.2, 12.7, 43);
+const bbox4326 = new BBox(CRS_EPSG4326, 11.9, 42.05, 12.95, 43.09);
+// const bbox4326 = new BBox(CRS_EPSG4326, 11.9, 42.2, 12.7, 43);
```
Comment (dgostencnik) [inline stories/landsat8.aws.stories.js]: Commented out code should be removed.

File: stories/index.stories.js [hunk 3]
```diff
@@ -262,35 +264,28 @@ export const S1GRDFindTiles = () => {
 
 export const findFlyovers = () => {
   const layer = new S2L2ALayer(instanceId, s2l2aLayerId);
-  const bbox = new BBox(CRS_EPSG4326, 11.9, 42.05, 12.95, 43.09);
 
   const wrapperEl = document.createElement('div');
-  wrapperEl.innerHTML = "<h2>findFlyovers</h2>";
+  wrapperEl.innerHTML = '<h2>findFlyovers</h2>';
```
Comment (dgostencnik) [inline stories/index.stories.js]: perhaps add S2/Sentinel 2 to heading  and function name ( both S1 and S2 are used in this story).

File: stories/s1grdiw.stories.js [hunk 4]
```diff
@@ -354,41 +350,34 @@ export const findFlyovers = () => {
...
-      } else {
-        text = tile[key];
-      }
-      li.innerHTML = `${key} : ${text}`;
-    }
-  });
-}
-
 export const findDatesEPSG4326 = () => {
   const bbox4326 = new BBox(CRS_EPSG4326, 11.9, 42.05, 12.95, 43.09);
```
Comment (dgostencnik) [inline stories/s1grdiw.stories.js]: This can be probably removed?

Comment (zcernigoj) [reply stories/s1grdiw.stories.js]: Thanks, removed.

File: stories/index.stories.js (hunk 3 above)
Comment (zcernigoj) [reply stories/index.stories.js]: Thanks, added.

File: stories/landsat8.aws.stories.js (hunk 2 above)
Comment (zcernigoj) [reply stories/landsat8.aws.stories.js]: Thanks, removed.

File: stories/index.stories.js (hunk 1 above)
Comment (zcernigoj) [reply stories/index.stories.js]: Thanks. I moved them into a separate file and imported them.

File: stories/byoc.stories.js [hunk 5]
```diff
@@ -1,15 +1,6 @@
-import moment from 'moment';
```
Comment (zcernigoj) [inline stories/byoc.stories.js]: Just a comment. In all stories I changed that the time parameters now use `Date`, not moment.

File: stories/landsat7.eocloud.stories.js [hunk 6]
```diff
@@ -1,5 +1,7 @@
 import moment from 'moment';
```
Comment (dgostencnik) [inline stories/landsat7.eocloud.stories.js]: moment leftover

Comment (zcernigoj) [reply stories/landsat7.eocloud.stories.js]: Thanks, removed

Comment (zcernigoj) [reply stories/landsat7.eocloud.stories.js]: Thanks, removed.

Comment (zcernigoj) [reply stories/landsat7.eocloud.stories.js]: I already merged as the changes were needed in the other branch

**Pull request merged**